cognite/neat/rules/importer/graph2rules.py

```diff
@@ -191,7 +181,15 @@ def _graph_to_data_model_dict(graph: Graph, max_number_of_instance: int = -1) ->
         Tuple of data model and prefixes of the graph
     """
     data_model: dict[str, dict] = {}
-    prefixes: dict[str, Namespace] = {}
+
+    prefixes: dict[str, Namespace] = {
+        "rdf": Namespace(str(RDF)),
+        "rdfs": Namespace(str(RDFS)),
+        "dct": Namespace(str(DCTERMS)),
+        "skos": Namespace(str(SKOS)),
+        "owl": Namespace(str(OWL)),
+        "xsd": Namespace(str(XSD)),
+    }
```

Comment:
Specific for this importer or defaults that should be available to all importers?